client/app/components/routes/operations/form/Operation.tsx

```diff
@@ -86,6 +89,20 @@ async function getOperation(id: number) {
   }
 }
 
+// 🛠️ Function to fetch an operator by ID
+async function getOperator(id: number) {
```

Comment:
We have almost the same function already in /components/routes/select-operator/form/ConfirmSelectedOperator.tsx, could reuse that one here or vice versa (this one looks like it has better error handling)